warehouse/clickhouse/clickhouse.go

```diff
@@ -378,6 +378,11 @@ func (ch *HandleT) DownloadLoadFiles(tableName string) ([]string, error) {
 	return fileNames, dErr
 }
 
+//Need to create a structure with delete parameters instead of simply adding a long list of params
+func (ch *HandleT) DeleteBy(tableNames []string, jobRunID string, sourceID string, taskRunID string) (success bool, err error) {
+	return false, fmt.Errorf("click err :not implemented")
```

Comment:
I would name it as `Not Implemented` to be generic with this kind of error. Wdyt ?

Reply:
Sure. Makes sense.

Reply:
Added the generic error.